packages/design-system/src/components/primitives/arrow-focus.stories.tsx

```diff
@@ -0,0 +1,108 @@
+import { StorySection } from "../storybook";
+import {
+  handleArrowFocus,
+  setArrowFocusRow,
+  setArrowFocusColumn,
+} from "./arrow-focus";
+
+const main = {
```

Comment:
I used to use this style initially but then realize if we don't wrap it into css() and don't use CSS type , you don't get the validation by ts

Reply:
This is stories so I didn't think to much about it. Does it matter in stories?

Reply:
yeah, idk